test/e2e/e2e_suite_test.go

```diff
@@ -61,7 +61,7 @@ const (
 	volcanoSchedulerManifestPath   = rootPath + "/dep-manifests/volcano-scheduler/" // all in one yaml of volcano-development.yaml
 	envUseExistingSchedulerPlugins = "USE_EXISTING_SCHEDULER_PLUGINS"
 	envUseExistingVolcanoScheduler = "USE_EXISTING_VOLCANO_SCHEDULER"
-	defaultSchedulerPluginsVersion = "v0.29.8"
+	defaultSchedulerPluginsVersion = "v0.31.1"
```

Comment:
```suggestion
	defaultSchedulerPluginsVersion = "v0.29.8"
```

This is the scheduler-plugins version, not k8s version.

Reply:
changed back